modules/programs/rofi-pass.nix

```diff
@@ -0,0 +1,32 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.programs.rofi.pass;
+in {
+  meta.maintainers = [ maintainers.seylerius ];
+
+  options.programs.rofi.pass = {
+    enable = mkEnableOption "Enable rofi integration with password-store";
+    root = mkOption {
+      type = types.str;
+      default = "";
+      description =
+        "Directory root of your password-store. Separate multiple roots with `:`, as `/foo/bar/baz:/path/to/root2`.";
+    };
+    extraConfig = mkOption {
+      type = types.str;
+      default = "";
+      description =
```

Comment:
add an example. Can you use a types.attrs with a lib.generators.toINI instead ?

Reply:
Not sure. The reason I originally didn't is that rofi-pass wants a pure-shell config file, not an INI, and I don't know how that will mesh.

Reply:
Even if it has to remain as a textual `extraConfig` it would be good to add an example. Just a line or two is enough.